py4DSTEM/process/rdf/amorph.py

```diff
@@ -3,7 +3,11 @@
 import py4DSTEM
 import scipy.io as sio
 from py4DSTEM.process.utils import print_progress_bar
-from py4DSTEM.process.utils.ellipticalCoords import *
+from py4DSTEM.process.utils.ellipticalCoords import * ## What else is used here? These fns have
+                                                      ## moved around some. In general, specifying
+                                                      ## the fns is better practice. TODO: change
```

Comment:
I left this because the amorph.py stuff was your code, and making a bunch of edits to that file seemed outside the scope of this PR, but thought it would be good to change this line from an `import *` to importing specific fns eventually.  So ya, I guess I would leave this for now, and also ya, it would be good to make the changes and remove the comment at some point